examples/reg-issue/src/utils/capitalizeSections.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * A function that takes in a string and capitalizes the first letter of each word.
+ * @author Dustin Newman
+ * @export
+ * @param {string} The string.
+ * @returns {string} A capitalized string.
+ */
+export default function capitalizeSections(str: string): string {
```

Comment:
Let's rename this to `capitalizeSection`, since we're only taking in one section at a time. Also let's rename `str` to `section` to make it clearer the string is the name of the section.